wsd/COOLWSD.cpp

```diff
@@ -3726,7 +3724,7 @@ int COOLWSD::innerMain()
     Anonymizer::mapAnonymized("contents", "contents");
 
     // Start the server.
-    Server->start();
+    Server->start(std::move(serverPort));
```

Comment:
If we want to enforce this, we want `start(std::shared_ptr<ServerSocket>&& serverSocket)`.

Reply:
Did that then